res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -155,25 +155,25 @@ var leds = {
     "hotCue2": 0x1c,
     "hotCue3": 0x1d,
     "hotCue4": 0x1e,
-    "Cue": 0x03,
+    "cue": 0x03,
```

Comment:
FYI, if you forget something in a commit and haven't added any new commits yet, `git commit --amend` is an easy way to overwrite the incomplete commit.